imsim/batoid_wcs.py

```diff
@@ -572,8 +554,17 @@ def makeWCSFactory(self, boresight, rotTelPos, obstime, band, camera='LsstCam',
             H2O_pressure = 1.0 # kPa
 
         # Finally, build the WCS.
-        return BatoidWCSFactory(boresight, rotTelPos, obstime, fiducial_telescope,
-                                wavelength, self.camera, temperature, pressure, H2O_pressure)
+        return BatoidWCSFactory(
+            boresight, obstime, telescope, wavelength, self.camera, temperature,
+            pressure, H2O_pressure
+        )
+
+    def quickFactory(self, boresight, rotTelPos, obstime, band, camera='LsstCam'):
+        from .telescope_loader import load_telescope
+        telescope = load_telescope(f"LSST_{band}.yaml", rotTelPos=rotTelPos)
+        return self.makeWCSFactory(
+            boresight, obstime, telescope, bandpass=band, camera=camera
+        )
 
 
 RegisterWCSType('Batoid', BatoidWCSBuilder())
```

Comment:
This should mention that it's associated with the Telescope input type now.
```
RegisterWCSType('Batoid', BatoidWCSBuilder(), input_type='telescope')
```
cf. http://galsim-developers.github.io/GalSim/_build/html/config_image.html#wcs-field